Formula/virtuoso.rb

```diff
@@ -38,7 +37,7 @@ class Virtuoso < Formula
 
   # Support OpenSSL 1.1
   patch do
-    url "https://sources.debian.org/data/main/v/virtuoso-opensource/7.2.5.1+dfsg-2/debian/patches/ssl1.1.patch"
+    url "https://sources.debian.org/data/main/v/virtuoso-opensource/7.2.6.1+dfsg-2/debian/patches/ssl1.1.patch"
```

Comment:
`curl: (22) The requested URL returned error: 404 `

For both previous and updated URLs.

Reply:
Maybe just delete the patch?
7.2.6+ has commits like:
- **Added initial support for OpenSSL 1.1.1** https://github.com/openlink/virtuoso-opensource/commit/713fa25b14457aa5127fac071830a2d20f4f968c
- **Removed support for OpenSSL 0.9.7 and earlier** https://github.com/openlink/virtuoso-opensource/commit/88c4eed7c4549f3162be4057b184b79cf231068d